code/modules/mob/living/living.dm

```diff
@@ -367,7 +367,10 @@
 
 //same as above
 /mob/living/pointed(atom/A as mob|obj|turf in view())
-	if(incapacitated())
+	var/obj/item/clothing/suit/straightjacket = src.get_item_by_slot(ITEM_SLOT_OCLOTHING)
```

Comment:
gotta give the variable an actual name, currently it's a /obj/item/clothing/suit variable that's named straightjacket
meaning literally any /suit item will prevent pointing